Generator/Generator/Arguments.swift

```diff
@@ -230,6 +232,8 @@ func generateCopies (_ args: [JGodotArgument]) -> String {
     return body
 }
 
+#if LEGACY_MARSHALING
+
```

Comment:
I also wonder if we could make this a runtime feature rather than a compile time feature.

The reason being, that way, if we find that for a specific method we need the legacy option, we could turn that on/off.

Reply:
Hm, perhaps we also could check if there is a way in SPM to propagate some `define` to dependency libraries?

Reply:
Ah, I misunderstood you. You probably meant in the scope of CodeGen.

Reply:
Done in [8e5edac](https://github.com/migueldeicaza/SwiftGodot/pull/533/commits/8e5edacf43bfbb383c48514a1eec1bc7f6c7c88b)

Reply:
Thank you!  This is brilliant!